script/rest/update-files.js

```diff
@@ -95,7 +97,7 @@ async function getBundledFiles(schemas) {
   }
 
   execSync(
-    `find ${TEMP_DOCS_DIR} -type f -name "*deref.json" -exec mv '{}' ${DEREFERENCED_DIR} ';'`
```

Comment:
>     `find ${TEMP_DOCS_DIR} -type f -name "*deref.json" -exec mv '{}' ${DOCS_DEREF_OPENAPI_DIR} ';'`
   )

   rimraf.sync(TEMP_DOCS_DIR)
  // When running in development mode, the the info.version
  // property in the dereferenced schema is replaced with the branch
   // name of the `github/github` checkout. A CI test
   // checks the version and fails if it's not a semantic version.
   for (const filename of schemas) {